### PR TITLE
App in meeting -nodejs

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**
Sample to add a application before, after and during a teams meeting -nodejs.
It covers the following functionality:

Meeting participants can collaborate through added aap in meeting.

### CODE REVIEW

**Review by v-chetsh — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@
 |8|App Localization | Microsoft Teams app localization using Bot and Tab |[View][app-localization#cs]  | [View][app-localization#js]
 |9|Details Tab in Meeting | Microsoft Teams meeting extensibility sample for iteracting with Details Tab in-meeting |[View][meetings-details-tab#cs]  |[View][meetings-details-tab#js]
 |10|App SSO | Microsoft Teams app SSO for Tab, Bot, ME - search, action, linkunfurl |[View][app-sso#cs] |[View][app-sso#js] 
+|10|App in meeting | This example shows app to add before, after and during a meting  ||[View][app-in-meeting#js]
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply README.md]: Updated

File: samples/app-in-meeting/nodejs/auth.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+var request = require('request');
+var graph = require('@microsoft/microsoft-graph-client');
+var Q = require('q');
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/auth.js:3]: Inherited from other approved samples need it for specific purpose

File: samples/app-in-meeting/nodejs/static/styles.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  }
+
+  .chatSection .task-status {
+    margin-left: 1rem;
+  }
+
+  .chatSection .task-description {
+    margin-left: 0rem;
+  }
+  .details {
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/static/styles.css]: Added

File: samples/app-in-meeting/nodejs/views/configure.ejs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        </header>
+        <script src="https://statics.teams.cdn.office.net/sdk/v1.10.0/js/MicrosoftTeams.min.js" integrity="sha384-6oUzHUqESdbT3hNPDDZUa/OunUj5SoxuMXNek1Dwe6AmChzqc6EJhjVrJ93DY/Bv" crossorigin="anonymous"></script>
+        <script>
+            function onClick() {
+                microsoftTeams.settings.setValidityState(true);
+            }
+            microsoftTeams.initialize();
+            microsoftTeams.appInitialization.notifySuccess();
+            microsoftTeams.settings.registerOnSaveHandler(function (saveEvent) {
+                var radios = document.getElementsByName('notificationType');
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/views/configure.ejs:34]: updated

File: samples/app-in-meeting/nodejs/views/index.ejs [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> App in a meeting Node js</title>
+</head>
+<body>
+    <div>
+        <h4>
+            App in a meeting node js sample
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/views/index.ejs]: Updated

File: README.md [hunk 6]
```diff
@@ -120,6 +120,7 @@
 |8|App Localization | Microsoft Teams app localization using Bot and Tab |[View][app-localization#cs]  | [View][app-localization#js]
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply README.md:120]: Added

File: samples/app-in-meeting/nodejs/views/appInMeeting.ejs [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+      <button onclick="sendTaskDetails()" class="send-button">Send Details</button>
+    </form>
+  </div>
+  <script>
+    var screenWidth = window.innerWidth;
+    var meetingId;
+    var tenantId;
+    var userId;
+    var userName;
+    $(document).ready(function () {
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/views/appInMeeting.ejs:57]: Added

File: samples/app-in-meeting/nodejs/Manifest/manifest.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "App In Meeting",
+    "full": "App In Meeting"
+  },
+  "description": {
+    "short": "App In Meeting",
+    "full": "This example shows app to add before, after and during a meeting"
```
[the remark this answers is not in the document]
Comment (v-kalc) [reply samples/app-in-meeting/nodejs/Manifest/manifest.json]: updated

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by AbhijeetThakur-msft — APPROVED** [review]
looks good!

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/app-in-meeting/nodejs/Manifest/manifest.json [hunk 9]
```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.9/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.9",
+  "version": "1.0.0",
+  "id": "<<Manifest-Id>>",
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/app-in-meeting/nodejs/Manifest/manifest.json]: changed

File: samples/app-in-meeting/nodejs/Manifest/manifest.json [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+        "team",
+        "groupchat"
+      ],
+      "context": [
+        "channelTab",
+        "privateChatTab",
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/app-in-meeting/nodejs/Manifest/manifest.json]: removed not required one's

File: samples/app-in-meeting/nodejs/Readme.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+![Share Icon](Images/share_icon.png)
+
+    - You can now interact with the app.
+
+
+## Features of this sample
+
+- App in stage view.
+
+![Stage View Screen](Images/stage_view.png)
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/app-in-meeting/nodejs/Readme.md]: done

File: samples/app-in-meeting/nodejs/Readme.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- Publicly addressable https url or tunnel such as [ngrok](https://ngrok.com/) or [Tunnel Relay](https://github.com/OfficeDev/microsoft-teams-tunnelrelay) 
+
+## Setup
+
+1. Run ngrok - point to port 3978
```
[the remark this answers is not in the document]
Comment (AbhijeetThakur-msft) [reply samples/app-in-meeting/nodejs/Readme.md]: done

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**